test/readme.md

```diff
@@ -729,6 +696,29 @@ source $DRIVERS_TOOLS/.evergreen/init-node-and-npm-env.sh
 bash .evergreen/run-deployed-gcp-kms-tests.sh
 ```
 
+
+### Deployed Atlas Tests
+
+#### Launching an Atlas Cluster
+
+Using drivers evergreen tools, run the `setup-atlas-cluster` script.  You must also set the CLUSTER_PREFIX environment variable.
+
+```bash
+CLUSTER_PREFIX=dbx-node-lambda bash ${DRIVERS_TOOLS}/.evergreen/atlas/setup-atlas-cluster.sh
+```
+
+The URI of the cluster is available in the `atlas-expansions.yml` file.
+
+#### Search Indexes
+
+1. Set up an Atlas cluster, as outlined in the "Launching an Atlas Cluster" section.
+2. Add the URI of the cluster to the environment as the MONGODB_URI environment variable.
+3. Run the tests with `npm run check:search-indexes`.
+
+#### Deployed Lambda Tests
+
+TODO
```

Comment:
@durran I don't actually know how to run these tests locally - maybe you could fill this section in?

Reply:
Oh boy it's been a long time. How about we create another ticket for that since it will take me some time to do it again.